cmd/util.go

```diff
@@ -291,11 +291,18 @@ func chooseStack(
 	return stack, nil
 }
 
+type projType string
+
+const (
+	pulumiAppProj    projType = "pulumi-app"
+	pulumiPolicyProj projType = "pulumi-policy"
+)
+
 // readProject attempts to detect and read the project for the current workspace. If an error occurs, it will be
 // printed to Stderr, and the returned value will be nil. If the project is successfully detected and read, it
 // is returned along with the path to its containing directory, which will be used as the root of the project's
 // Pulumi program.
-func readProject() (*workspace.Project, string, error) {
+func readProject(projType projType) (*workspace.Project, string, error) {
```

Comment:
Can we update the doc comment here to indicate what the significance of the `projType` parameter is? How does that change the behavior of the function, if at all?